packages/xo-web/src/xo-app/proxies/index.js

```diff
@@ -25,17 +30,187 @@ import {
 
 import Page from '../page'
 
+const Label = ({ children, ...props }) => (
+  <label {...props} style={{ cursor: 'pointer' }}>
+    <strong>{children}</strong>
+  </label>
+)
+
 const _deployProxy = () =>
   form({
-    render: ({ onChange, value }) => (
-      <SelectSr onChange={onChange} value={value} required />
-    ),
+    defaultValue: {
+      dns: '',
+      gateway: '',
+      ip: '',
+      netmask: '',
+      networkMode: 'dhcp',
+    },
+    component: decorate([
+      provideState({
+        effects: {
+          onSrChange(_, sr) {
+            this.props.onChange({
+              ...this.props.value,
+              sr,
+            })
+          },
+          onChange(_, { target: { name, value } }) {
+            this.props.onChange({
+              ...this.props.value,
+              [name]: value,
+            })
+          },
+        },
+        computed: {
+          idDhcpInput: generateId,
+          idDnsInput: generateId,
+          idGatewayInput: generateId,
+          idIpInput: generateId,
+          idNetmaskInput: generateId,
+          idSelectSr: generateId,
+          idStaticIpInput: generateId,
+        },
+      }),
+      injectState,
+      injectIntl,
+      ({ effects, state, value, intl: { formatMessage } }) => (
+        <Container>
+          <SingleLineRow>
+            <Col mediumSize={4}>
+              <Label htmlFor={state.idSelectSr}>{_('destinationSR')}</Label>
+            </Col>
+            <Col mediumSize={8}>
+              <SelectSr
+                id={state.idSelectSr}
+                onChange={effects.onSrChange}
+                required
+                value={value.sr}
+              />
+            </Col>
+          </SingleLineRow>
+          <SingleLineRow className='mt-1'>
+            <Col mediumSize={4}>
+              <Label htmlFor={state.idDhcpInput}>{_('dhcp')}</Label>
+            </Col>
+            <Col mediumSize={8}>
+              <input
+                checked={value.networkMode === 'dhcp'}
+                id={state.idDhcpInput}
+                name='networkMode'
+                onChange={effects.onChange}
+                type='radio'
+                value='dhcp'
+              />
+            </Col>
+          </SingleLineRow>
+          <SingleLineRow>
+            <Col mediumSize={4}>
+              <Label htmlFor={state.idStaticIpInput}>{_('staticIp')}</Label>
+            </Col>
+            <Col mediumSize={8}>
+              <input
+                checked={value.networkMode === 'staticIp'}
+                id={state.idStaticIpInput}
+                name='networkMode'
+                onChange={effects.onChange}
+                type='radio'
+                value='staticIp'
+              />
```

Comment:
I think we can remove this by switching to static mode if the user input a static IP.

This simplify the form and require less interaction from the user.

Reply:
How it will be in term of `UI`? Did other fields will be always displayed? Is it clear to users that the default network configuration mode is `DHCP`?

Reply:
Ideally, a input select with DHCP by defaut would make more sense. If the use open it and select "Static", then all network fields will be displayed. For me it's better than a radio button.

Plus you can name the section "Networking" (instead of two names DHCP and Static)

Reply:
Something like that:

![74825522-622b3a80-530a-11ea-928b-37f089c483b8](https://user-images.githubusercontent.com/1241401/74829969-707d5480-5312-11ea-9516-e6cd204fc126.png)

Reply:
Copy, better thank you :+1:

Reply:
Don't forget to update the screenshots in the PR :) (when you have the definitive UI).

Thanks!

Reply:
Done :+1: